src/openassetio-python/tests/package/pluginSystem/test_hybridpluginsystemmanagerimplementationfactory.py

```diff
@@ -0,0 +1,1637 @@
+
+    def test_when_single_factory_matches_then_uses_that_factorys_interface(
+        self, mock_logger, factory_a, factory_b, the_plugin_identifier
+    ):
+        factory_a.mock.identifiers.return_value = ["a", the_plugin_identifier]
+        factory_b.mock.identifiers.return_value = ["b"]
+
+        factory = HybridPluginSystemManagerImplementationFactory(
+            [factory_a, factory_b], mock_logger
+        )
+
+        actual_manager_interface = factory.instantiate(the_plugin_identifier)
+
+        factory_a.mock.instantiate.assert_called_once_with(the_plugin_identifier)
+        assert not factory_b.mock.instantiate.called
+        assert actual_manager_interface is factory_a.mock.instantiate.return_value
+
+    def test_when_many_factories_provided_and_one_matches_then_uses_that_factorys_interface(
+        self,
+        mock_logger,
+        create_mock_manager_interface,
+        factory_a,
+        factory_b,
+        the_plugin_identifier,
+    ):
+        factory_a.mock.identifiers.return_value = ["a"]
+        factory_b.mock.identifiers.return_value = ["b"]
+        factory_c = MockManagerImplementationFactory(mock_logger)
+        factory_c.mock.identifiers.return_value = ["c"]
+        factory_d = MockManagerImplementationFactory(mock_logger)
+        factory_d.mock.identifiers.return_value = [the_plugin_identifier]
+        factory_d.mock.instantiate.return_value = create_mock_manager_interface()
+        factory_e = MockManagerImplementationFactory(mock_logger)
+        factory_e.mock.identifiers.return_value = ["e"]
+
+        factory = HybridPluginSystemManagerImplementationFactory(
+            [factory_a, factory_b, factory_c, factory_d, factory_e], mock_logger
+        )
+
+        actual_manager_interface = factory.instantiate(the_plugin_identifier)
+
+        assert not factory_a.mock.instantiate.called
+        assert not factory_b.mock.instantiate.called
+        assert not factory_c.mock.instantiate.called
+        factory_d.mock.instantiate.assert_called_once_with(the_plugin_identifier)
+        assert not factory_e.mock.instantiate.called
+
+        assert actual_manager_interface is factory_d.mock.instantiate.return_value
+
+    def test_when_multiple_factories_match_then_hybrid_interface_used(
+        self, mock_logger, factory_a, factory_b, the_plugin_identifier
+    ):
+        factory = HybridPluginSystemManagerImplementationFactory(
+            [factory_a, factory_b], mock_logger
+        )
+
+        actual_manager_interface = factory.instantiate(the_plugin_identifier)
+
+        factory_a.mock.instantiate.assert_called_once_with(the_plugin_identifier)
+        factory_b.mock.instantiate.assert_called_once_with(the_plugin_identifier)
+        assert isinstance(actual_manager_interface, ManagerInterface)
+        assert actual_manager_interface is not factory_a.mock.instantiate.return_value
+        assert actual_manager_interface is not factory_b.mock.instantiate.return_value
```

Comment:
Yeah its not very satisfying, since there is no way to actually detect that `actual_manager_interface` is a `HybridManagerInterface`.  The only way is through usage, so in that respect the missing coverage is added by the `Test_HybridPluginSystemManagerImplementationFactory_ManagerInterface` tests, below.